docs/xxjitservertimeout.md

```diff
@@ -26,20 +26,23 @@
 
 This option specifies the socket timeout for JITServer communication.
 
-It can be specified on both the server and the client.
+You can specify this option on both the server and the client sides.
 
 ## Syntax
 
         -XX:JITServerTimeout=<timeout_ms>
 
 | Setting                 | Effect | Default                                                                            |
 |-------------------------|--------|:----------------------------------------------------------------------------------:|
-|`-XX:JITServerTimeout`           | Set the timeout value in milliseconds for socket operations | **Note 1**                                                                                    |
+|`-XX:JITServerTimeout`           | Set the timeout value in milliseconds for socket operations | 30000 ms for the JITServer process and 10000 ms when OpenJ9 is launched as a client VM                                                                                    |
 
-**Note 1:** The timeout default is 30000 ms for the JITServer process, and 10000 ms when OpenJ9 is launched as a client VM.
 
 ## Explanation
 
 The default values should be reasonable for most use cases.
 
```

Comment:
I am thinking of removing this text because it's not really an explanation and does not have much value.

Reply:
Thanks, I removed the text.